src/GeoJson/Exception/UnserializationException.php

```diff
@@ -1,18 +1,19 @@
 <?php
 
+declare(strict_types=1);
+
 namespace GeoJson\Exception;
 
-class UnserializationException extends \RuntimeException implements Exception
+use RuntimeException;
+
+class UnserializationException extends RuntimeException implements Exception
 {
     /**
      * Creates an UnserializationException for a value with an invalid type.
      *
-     * @param string $context
      * @param mixed $value
-     * @param string $expectedType
-     * @return UnserializationException
      */
-    public static function invalidValue($context, $value, $expectedType)
+    public static function invalidValue(string $context, $value, string $expectedType): self
```

Comment:
Outside the scope of this PR, but we could consider using [`get_debug_type`](https://www.php.net/get_debug_type) in the error message below (at the expense of requiring `symfony/polyfill-php80` for PHP 7.4).

Reply:
It's just another commit (which I just pushed 💪) - if you think it should be done in a 2.x release, I'll remove it again.

Reply:
No objections to including that in 1.1. Thanks!